py/torch_tensorrt/_enums.py

```diff
@@ -107,7 +107,7 @@ def _from(
                 return dtype.f16
             elif t == trt.float32:
                 return dtype.f32
-            elif trt.__version__ >= "7.0" and t == trt.bool:
```

Comment:
Why are you removing the version check?

Reply:
This version check actually fails. If trt.__version == 10.0.0b6, then `trt.__version__ >= "7.0"` is False (which should actually be true) and hence the trt.bool type wouldn't be returned which results in type errors.
We also don't need these version checks because we only support strict TRT versions.